tool-bar.ios.ts

```diff
@@ -111,7 +111,7 @@ export class ToolBar extends ToolBarDefinition {
         let navBarHeight = 0;
 
         if (heightMode != layout.EXACTLY) {
-            let toolbarSize = this._ios.intrinsicContentSize();
+            let toolbarSize = this._ios.intrinsicContentSize;
             newHeight = toolbarSize.height;
```

Comment:
A quick test with XCode 7.3 and iOS9 shows that this change won't work on that configuration. It won't crash but it seems that the size won't be returned. On  XCode 8 + iOS10 the old version will crash. Unfortunately I'm unable to test this further right now and will use a workaround for now.